pokedex_async_redux/lib/features/pokemon_details/widget/pokemon_general.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:pokedex_async_redux/utilities/colors.dart';
+import 'package:flutter/material.dart';
+
+class GeneralDetailsView extends StatelessWidget {
+  const GeneralDetailsView({
+    required this.detailName,
+    required this.detailValue,
+    required this.detailType,
+    Key? key,
+  }) : super(key: key);
+
+  final double detailValue;
+  final String detailName;
+  final String detailType;
+
+  @override
+  Widget build(BuildContext context) {
+    return Chip(
+      label: Text(
+        "$detailName: ${detailValue.toString().split(".").first} $detailType",
```

Comment:
kindly wrap this in a new variable with better name, make it readily available before the `return` statement 🙂